core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -814,6 +814,8 @@ private[spark] class TaskSchedulerImpl(
   // By default, rack is unknown
   def getRackForHost(value: String): Option[String] = None
 
+  def getRacksForHosts(values: List[String]): List[String] = Nil
```

Comment:
shouldn't the default be `values.map(getRackForHost)`?  otherwise this will be wrong in a subclass which only overrides `getRackForHost`

Reply:
Good finding. I will fix it.